### PR TITLE
update of integration test framework, with many new tests

### PR DESCRIPTION
**Pull request by francois-vincent**

### CODE REVIEW

File: integration_tests/conftest.py [hunk 1]
```diff
@@ -6,15 +6,17 @@
 from docker import PlatformManager, DeployedPlatformManager
 from fabric_integration import FabricManager
 
+DEFAULT_DEBIAN = 'debian8'
```
Comment (antoine-de) [inline integration_tests/conftest.py]: why do you need to specify the debian version ?

Comment (francois-vincent) [inline integration_tests/conftest.py]: because I don't want to run the tests on all debian versions, it does not make sense

Comment (antoine-de) [inline integration_tests/conftest.py]: I don't understand, the tests are run in a docker no ?

File: integration_tests/README.md [hunk 2]
```diff
@@ -3,19 +3,21 @@ Integration tests project for https://github.com/CanalTP/fabric_navitia
 
 How to launch (as a fabric_navitia subproject):
 - cd tests_integration
-- copy navitia packages here
+- copy navitia packages here (mind your debian version !)
```
Comment (antoine-de) [inline integration_tests/README.md]: i really don't understand this debian version stuff, all the tests are launched in docker no ?

File: integration_tests/fabric_integration.py [hunk 3]
```diff
@@ -32,6 +32,13 @@
     fabric_tasks = utils.Command('fab no_such_task').stdout_column(0, 2)
 
 
+# LOG_PARAMIKO = 'paramiko.log'
+LOG_PARAMIKO = None
```
Comment (antoine-de) [inline integration_tests/fabric_integration.py]: what is this ?

File: integration_tests/conftest.py (hunk 1 above)
Comment (francois-vincent) [inline integration_tests/conftest.py]: yes but most of the time, you want to go as fast as possible (at least I do)
and if you want to run full tests on all possible cases, you run them twice with --distri=debian7 and you're done
I think that the most common use case must use the shortest command line
And if it appears, later that we should change this behaviour, thern we'll change it, it's easy

File: integration_tests/fabric_integration.py (hunk 3 above)
Comment (francois-vincent) [inline integration_tests/fabric_integration.py]: I've been using it to show paramiko logs, may be usefull to help debug

Comment (antoine-de) [inline integration_tests/fabric_integration.py]: I'm not a fan of useless lines of codes like this. 

If you need to keep it, at least put a comment around to remember why it's here :wink:

File: integration_tests/test_tasks/test_duplicated.py [hunk 4]
```diff
@@ -19,6 +25,9 @@ def test_upgrade_kraken(duplicated):
         fabric.execute_forked(
             'tasks.upgrade_kraken', kraken_wait=False, up_confs=False, supervision=False)
 
+    if SHOW_CALL_TRACKER_DATA:
+        from pprint import pprint
+        pprint(dict(data()))
```
Comment (antoine-de) [inline integration_tests/test_tasks/test_duplicated.py]: what is data() ?

I can't find the definition

Comment (francois-vincent) [inline integration_tests/test_tasks/test_duplicated.py]: it comes out of the context manager.

Comment (antoine-de) [inline integration_tests/test_tasks/test_duplicated.py]: And what is it exactly ?

**Comment by antoine-de** [comment]
I'm honestly not capable to review this completely, but I don't think anyone in the team is, so I merge it

**Pull request merged**